sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -110,10 +110,11 @@ interface Qonversion {
     /**
      * Add property value for the current user to use it then for segmentation or analytics
      * as well as to provide it to third-party platforms.
-     * This method consumes both defined and custom user properties. For defined properties
-     * use [UserProperty.code] as the key. In order to pass single property
-     * consider using [setCustomUserProperty] method for the custom keys and [setUserProperties]
-     * for defined ones.
+     * This method consumes both defined and custom user properties. Consider using
+     * [UserPropertiesBuilder] to prepare a properties map. You are also able to create it
+     * on your own using [UserProperty.code] as the key for Qonversion defined properties.
+     * In order to pass single property consider using [setCustomUserProperty] method for
+     * the custom keys and [setUserProperties] for defined ones.
```

Comment:
I'm not sure, but I think you mean `and [setUserProperty] for defined ones.` not `setUserProperties`